constants/styles.js

```diff
@@ -17,6 +17,7 @@ export const colors = {
   deep_saffron: '#F89235',
   transparentWhite: 'rgba(255, 255, 255, 0.9)',
   columbia_blue: '#c0d9e1',
+  mgh_blue: '#089ca4',
```

Comment:
do we need to add a new hex color for this feature?

Reply:
I think so, the other hex colors are used in the charts, but the app doesn't have a theme or color scheme. This is a color for the button that submits the filters. Would you like me to remove it?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/248/commits/2467f5a77a481233a7e141d858b7cd0668600c38